spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe User do
+  describe 'Associations' do
+    it { should have_many(:manifests) }
+  end
+
+  describe 'cdx_user_id' do
+    it 'is always required' do
+      user = create(:user)
+      expect(user.cdx_user_id).to_not be_nil    
```

Comment:
ah yes there is a weird bug in shoulda https://github.com/thoughtbot/shoulda-matchers/blob/585f651460026045f590ea27e14f6c5f09e8e6c4/lib/shoulda/matchers/active_model/validate_uniqueness_of_matcher.rb#L28

related to combo presence and uniqueness validation,s should be able to do:
```
subject { build(:user) }
it { should validate_uniqueness_of(:cdx_user_id) }
```

not sure what the other error is for the presence validation. if this is too much of a pain we can just skip it for now